README.md

```diff
@@ -133,33 +132,55 @@ mongo --version
 
-Getting a local instance of Quill up and running takes less than 5 minutes! Start by setting up the database. Ideally, you should run MongoDB as a daemon with a secure configuration (with most linux distributions, you should be able to install it with your package manager, and it'll be set up as a daemon). Although not recommended for production, when running locally for development, you could do it like this
+We use `dotenv` to keep track of environment variables, so be sure to stop tracking the `.env` file in Git:
 
 ```bash
-mkdir db
-mongod --dbpath db --bind_ip 127.0.0.1
+git update-index --assume-unchanged .env
 ```
 
-Install the necessary dependencies:
+After doing this, fill in the environment variables in the `.env` before running Quill.
+
+### Local Deployment
+
+#### MongoDB
+
+Ideally, you should run MongoDB as a daemon with a secure configuration (with most linux distributions, you should be able to install it with your package manager, and it'll be set up as a daemon). Although not recommended for production, when running locally for development, you could do it like this
 
 ```bash
-npm install
+mkdir db
+mongod --dbpath db --bind_ip 127.0.0.1
 ```
 
-We use `dotenv` to keep track of environment variables, so be sure to stop tracking the `.env` file in Git:
+#### SMTP
+
+This step is optional and is only required if you want to test the email-related functionality of Quill. The easiest option is to use the SMTP server provided by your personal email (Gmail, Outlook, etc.). Look for the documentation about SMTP for your respective email and fill in the values in the `.env` accordingly. Be warned that sending many emails this way is not recommended and this method should only be used for testing.
+
+#### Quill
+
+Install the necessary dependencies:
 
 ```bash
-git update-index --assume-unchanged .env
+npm install
 ```
 
-Edit the configuration file in `.env` for your setup, and then run the application:
+Ensure you have filled in the `.env` according to your setup, and then run the application:
 
 ```bash
 gulp server
 ```
+
 ### Deploying for your hackathon
+
+#### MongoDB
+
 The database can either be hosted with [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) or on your own server. Atlas will generally be easier to set up and should be the preferred choice unless you are familiar with administering your own server. A guide to setting up Atlas can be found [here](https://docs.atlas.mongodb.com/getting-started/). Note that the URI for the database (which must be specified in `.env`) will be different depending on where your database is hosted. 
 
+#### SMTP
+
+A dedicated SMTP provider is absolutely required if you want Quill to work for your hackathon. There are several providers available (Mailgun, Sendgrid, and others). After setting this up, fill in the `.env` with the values that your provider gives you.
```

Comment:
Maybe link out to the providers we name dropped? 

https://www.mailgun.com
https://sendgrid.com

Also, it's worth noting that both of them have deals they offer free plans via the Github Student pack:
https://sendgrid.com/blog/sendgrid-accelerate-edu/
https://www.mailgun.com/github-students/